### DISCUSSION TITLE
Custom Front / Rear ports

### DISCUSSION THREAD

**Discussion by vbarelds** [netbox-community/netbox#7782] [Q&A] [answered]
Hi everyone,

In one of our racks there is a device that receives multiple video inputs and combines certain inputs to different outputs.
This device uses DVI connection to do this.

In Netbox it it possibile to create ports with the names DVI 0/1, 0/2 etc.
But since the correct port type can't be chosen my "source of truth" is incorrect.

I tried to add ports in the file **choices.py** but that doesn't seem to do anything.

I attached a image as an example.
![example1](https://user-images.githubusercontent.com/93971030/140885549-b47e789b-c85f-4396-a7dc-003dc8fc02ae.jpg)

**Comment by candlerb** [comment, marked as answer]
The port types are not user-settable.  For interfaces there is "Other", but this doesn't seem to be available for front/rear ports; you could raise a feature request for this.

**Reply by vbarelds** [reply]
Thank you for your answer. 
I will raise a feature request for this.
